dist/api_data.py

```diff
@@ -1208,6 +1209,37 @@ def __ge__(self, other):
         characters are hexadecimal encoded.  These formats are compatible
         with the @ref util_dump and @ref util_load commands''',
         choices=['hex', 'json', 'print']),
+    Config('incremental', '', r'''
+        configure the cursor for block incremental backup usage. These formats
+        are only compatible with the backup data source; see @ref backup''',
+        type='category', subconfig=[
+        Config('enabled', 'false', r'''
+            whether to configure this backup as an incremental starting point''',
```

Comment:
"an incremental starting point" -> "the starting point for a subsequent incremental backup" ?

Reply:
Fixed.